wordpress-mobile/WordPress-iOS
# Feature/7675 social login service call

Refs #7675 
This PR adds the service and networking layer to support social login.  It also wires up the google sign in button to test the simplest login scenario.

This PR does not address cases where: 
- there is no matching WPcom account, 
- an matching wpcom account is found but is not already connected,
- 2fa is enabled.
These cases will be addressed in subsequent PRs.

To test:
Use the Google sign in button to login to a WPcom account that has already been connected to a Google profile. Confirm you can login successfully in this scenario.

Confirm that the google button is correctly positioned on the iPad.

Needs review: @nheagy
